tests/MassTransit.Tests/MessageUrnSpecs.cs

```
@@ -45,5 +73,29 @@ class X
 
     public class G<T>
     {
+    }
+
+    [MessageUrn("MyCustomName")]
+    public class Attributed
+    {
+
+    }
+
+    [MessageUrn(null)]
+    public class AttributedNull
+    {
+
+    }
+
+    [MessageUrn("")]
+    public class AttributedEmpty
+    {
+
+    }
+
+    [MessageUrn("\t\t   ")]
```

Yeah, I'd expect these to throw some type of exception about being invalid.